### ISSUE TITLE
Low Lighthouse score of Meshery.io 

### ISSUE THREAD

**Issue by Udit-takkar** [meshery/meshery.io#973]
### Description
meshery.io is failing the Core Web Vitals Assessment

Mobile performance score is between 20-40

Check here:- https://pagespeed.web.dev/report?url=https%3A%2F%2Fmeshery.io%2F&form_factor=mobile

### Expected Behavior
The landing page should pass the Core Web Vitals Assessment

### Screenshots

<img width="968" alt="Screenshot 2023-01-15 at 9 32 57 PM" src="https://user-images.githubusercontent.com/53316345/212552032-fb5b634f-6f5f-44d6-962b-23ff8055297f.png">

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

**Comment by Yashsharma1911** [comment]
Hey @Udit-takkar, do you have any suggestions on how we can improve it

**Comment by Udit-takkar** [comment]
@Yashsharma1911 will have to test a lot of things with lighthouse report and use some concepts learned from here https://github.com/meshery/meshery/pull/3572#event-8239131553. Some things like lazy loading below the fold images, using images of webp quality, properly loading the third party code, preloading fonts ... etc.

**Comment by Yashsharma1911** [comment]
Hey, @Udit-takkar are you still working on this?

**Comment by saisuvanth** [comment]
Hey @Yashsharma1911 , I have some ideas on how to make it better by reducing render blocking time. Can I work on this issue?

**Comment by Udit-takkar** [comment]
@saisuvanth yeah go ahead

**Comment by iArchitSharma** [comment]
@Udit-takkar are you still working on this issue?

**Comment by iArchitSharma** [comment]
Reanalyzed the assessments; improvements have been made.
![image](https://github.com/meshery/meshery.io/assets/74408634/9965802e-d332-45fc-91c1-2aa922acbb1b)

**Comment by Savio629** [comment]
Is it possible to increase the performance more?
@iArchitSharma

**Comment by iArchitSharma** [comment]
@Savio629, yes, it can be, check the recommendations in the Opportunities section of the results

**Comment by Akshun-01** [comment]
Hello, I would like to give this one a try!

**Comment by Yashsharma1911** [comment]
Please go ahead, also try sharing your progress as you go so that others can help you if needed ^^

**Comment by Akshun-01** [comment]
Hey, Just dropping the update on this.
### Changed the .gif files to .webp format.
( I tried webm and mpeg4 as it suggested but they were not working so I used webp instead)

This is the before and after change : 
 ### Before
![image](https://github.com/meshery/meshery.io/assets/93793691/531177b2-4c57-4685-8b9f-309699c1ebcc)

### After (best practice is low because of using local host instead of https)
![image](https://github.com/meshery/meshery.io/assets/93793691/f300c0ba-508a-4018-a884-66cf8b4592b5)


Should I start a draft PR, or make a PR after making considerable changes? @Yashsharma1911

**Comment by Yashsharma1911** [comment]
Hey this is good 👏, yes please raise a draft PR so that others can provide a review as you go.

**Comment by tayyab-ilyas** [comment]
Hi does this issue still needs to be addressed ? I ran a lighthouse test and the performance metric is currently at 53. I'd like to give this a go and target to touch 80-ish.

**Comment by Yashsharma1911** [comment]
Hey yes this is still an area for help, and target sounds good too, you can work on it, also as you go and explore ways to increase lighthouse score I'll suggest to create draft PR so that maintainers can provide you feedbaxk along the way

**Comment by tayyab-ilyas** [comment]
Hi @Yashsharma1911 I tried a bunch of stuff like deferring scripts, preloading CSS, and even tested out WebP format for assets but still no significant improvements in performance. I'll try to work on some other issue now. I hope that I didn't waste your time here. thanks!

**Comment by shubhusion** [comment]
I would like work on this issue

**Comment by shubhusion** [comment]
@Yashsharma1911 please assign this issue to me , I would like to give it a try

**Comment by akshansh-modi** [comment]
checked it out, i would like to work on this  
![image](https://github.com/user-attachments/assets/2a12420a-995a-406e-b4fa-54d3dbd4ef17)

**Comment by sudhanshutech** [comment]
Hey @shubhusion thanks for jumping in. Yes please go ahead just to be ensure it would be thing to discuss also while you will fixing this issue that what are the best practices so please regularly share your progress if you would. And @akshansh-modi please you also chime in with @shubhusion to help if you would.

**Comment by shubhusion** [comment]
### Results After Improving the Accessibility of the Website

![image](https://github.com/user-attachments/assets/77edf21e-8988-4745-9d7d-1f28436f200b)

The accessibility score has improved from 74 to 95.
